app/models/miq_queue.rb

```diff
@@ -364,6 +364,7 @@ def deliver(requester = nil)
       _log.error("#{MiqQueue.format_short_log_msg(self)}, #{message}")
       status = STATUS_EXPIRED
     rescue Exception => error
+      raise if error.class.name == "IRB::Abort"
```

Comment:
Seems reasonable to me as well.  Didn't suggest that incase, now that we are 7+ years later with this change in place, something else isn't depending on it being `rescue Exception`...

But I agree that this "seems" like the more correct way of handling this.